Makefile

```diff
@@ -203,8 +203,8 @@ CIRRUS_BLACK_FIX = black -l 90 cirrus/server
 CIRRUS_RUFF_CHECK = ruff cirrus/server
 CIRRUS_RUFF_FIX = ruff --fix cirrus/server
 CIRRUS_PYTEST = pytest cirrus/server --cov=cirrus
-
-
+CIRRUS_PYTHON_TYPECHECK = pyright -p cirrus/server
+CIRRUS_PYTHON_TYPECHECK_CREATESTUB = pyright -p cirrus/server --createstub cirrus
```

Comment:
Where do we plan to use this?

Reply:
I am using cirrus_python_typecheck in the make check command to verify the strict type, and the second command is to generate the stubs for the python code, I have ran it initially but later on can also be added a part of check command to make sure ours stubs are up to date with the code, right now just added as a make command